spark/src/test/scala/geotrellis/spark/io/LayerUpdateSpaceTimeTileFeatureTests.scala

```diff
@@ -0,0 +1,113 @@
+package geotrellis.spark.io
+
+import geotrellis.proj4.LatLng
+import geotrellis.raster._
+import geotrellis.raster.testkit._
+import geotrellis.spark._
+import geotrellis.spark.io.avro.codecs._
+import geotrellis.spark.io.json._
+import geotrellis.spark.testkit._
+import geotrellis.spark.tiling._
+import geotrellis.spark.util._
+import geotrellis.util._
+import geotrellis.vector._
+
+import org.apache.spark.rdd._
+import org.joda.time.DateTime
+
+trait LayerUpdateSpaceTimeTileFeatureTests
```

Comment:
I named this one after the file `.../spark/io/AllOnesTestTileTests.scala` from which it was copied (same is true for `.../spark/io/CoordinateSpaceTimeTileFeatureTests.scala` and `.../spark/io/LayerUpdateSpaceTimeTileFeatureTests.scala`).  I can change the names of one or both of these sets of tests, if desired.